### DISCUSSION TITLE
How to use 100% of the CPU

### DISCUSSION THREAD

**Discussion by mr-qansuh** [staxrip/staxrip#529] [Q&A]
How to make Staxrip use 100% of the CPU

**Comment by stax76** [comment]
You can find some info here:

https://github.com/staxrip/staxrip/wiki/Usage#parallel-processing

**Comment by ghost** [comment]
Yes please, always check the Wiki, because this question has already been asked especially by people with Ryzen processors.
- solution 1: run jobs in parallel (if you have several jobs, but you can't if you only have 1 job)
- solution 2: if you are encoding with x265, use chunks ( 2 or 3). Chunks means that the video is divided to n chunks and every chunk is encoded separately, at the same time. Chunks is found in x265 options > Input/Output > Chunks.

This is a copy of a text a wrote in another place (it was about Ryzen processors) :
What you must know:
- If you are encoding 4K, you'll get 100% CPU all right.
- If you're encoding FHD, then you'll not get 100% CPU and the solution for you in that case is to use the "Chunks" feature in x265 options > Input / Output > "Chunks", set it to 2 or 3.
- If you're encoding HD or less, then you'll need to set the Chunks to 4 or 5 and need to increase the max number of parallel processes to 5 in the settings > System (and leave it to 5 afterwards)

But if you use x264, you can only run jobs in parallel, since x264 chunks are not supported...

**Comment by fmohican** [comment]
atlast for me, on x264, i use vapoursynth instead of avisynth, and always i got over 70% on Ryzen 9 3950x

X264 = Slower + CRF 16

[proof](https://img.aniro.me/images/2021/03/01/1b195a84eab2111f2ec9f25b56fc7d0a.png)

**Reply by ZeroCool22** [reply]
How much time to encode a movie on x265 preset slow with that CPU?

**Reply by fmohican** [reply]
I don't know. But i can try if you're interesting, can you contact me via Discord? ( Fmohican#6583 )

**Reply by ZeroCool22** [reply]
Done. :)
